web/src/main/java/org/fao/geonet/monitor/link/LinkInfo.java

```diff
@@ -0,0 +1,91 @@
+package org.fao.geonet.monitor.link;
+
+import org.jdom.Element;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class LinkInfo {
```

Comment:
I'm finding it a bit difficult to understand the overall algorithm of checking in this class...